packages/open-next/src/http/openNextResponse.ts

```diff
@@ -92,8 +82,8 @@ export class OpenNextNodeResponse extends Transform implements ServerResponse {
   }
 
   get finished() {
-    return Boolean(
-      this.writableFinished && this.responseStream?.writableFinished,
+    return (
+      this.writableFinished && (this.responseStream?.writableFinished ?? true)
```

Comment:
I'm not sure this one is correct actually. 
Your fix is good in case we don't provide a `ResponseStream`, but in case we do, we may end up in a situation where `OpenNextNodeResponse` is finished, but not `this.responseStream`. I'm not 100% sure of that though and not sure what could be the consequences either.

I think we could replace it with 
```ts
this.responseStream ? this.responseStream.writableFinished : this.writableFinished
```

Reply:
swapped for your suggestion, thanks!